loxigen.py

```diff
@@ -103,17 +103,18 @@ def process_input_file(filename):
 
     # Parse the input file
     try:
-        with open(filename, 'r') as f:
+        # handle other encodings without throwing UnicodeDecodeError
+        with open(filename, encoding='ascii', errors='surrogateescape') as f:
```

Comment:
would it be possible to use utf-8? seems like `surrogateescape` might mask some errors.

Reply:
do you have any suggestions on the error handling?  it looks like standard-1.5 doesn't throw an error if i use "open(filename, encoding='utf-8', errors='replace')", but i'm wondering if there is something more appropriate.

Reply:
I’d say we probably don’t need much error handing? We control the input file and it should be valid utf-8. It seems ok to just let the exception fly if that’s violated because that would only happen if somebody adds non-UTF-8 code to our open flow format definitions?

Reply:
unfortunately parsing the current standard-1.5 will die with a UnicodeDecodeError if the error handler isn't specified.  i tried manually stripping out the offending characters, but it was a long slog (more than 20, but don't remember exactly)

Reply:
Oh wow, that seems brøken... maybe would be better to clean up the openflow input file rather than make the parser ignore brokenness, but we could totally just file a Jira and do that in a separate PR?

Reply:
ok, will change to "open(filename, encoding='utf-8', errors='replace')" for now, and fix the error handling and standard-1.5 input file in another PR.

just in case you're wondering, the failing lines seem to be comments; 
for example:
"Tüm sürümler için ayný olsun diye multipart yerine stats"
which translates to 
"Stats instead of multipart to be the same for all versions"